test/TensorFlow/control_flow.swift

```diff
@@ -87,9 +87,9 @@ public func weighPetOnlyDefault(pet: Pet) {
 // CHECK:           return
 // CHECK-NEXT:  ---- END OF ANALYSIS STATE FOR FUNCTION
 
-public func testCondBranch(_ a: Bool) { // expected-warning {{'a' implicitly copied to the accelerator}}
+public func testCondBranch(_ a: Bool) {
```

Comment:
should we duplicate one such scalar warning test, and turn on TFWarnScalarTransfer so that there is some coverage there?

Reply:
should this PR include `diagnostics_scalar_transfers.swift`?

Reply:
oops I forgot to `git add` it. will do that